modules/@apostrophecms/submitted-draft/ui/apos/components/AposSubmittedDraftAdminBarIcon.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <arrow-expand-right-icon
```

Comment:
This should be an `<AposIndicator />`, use the tray-full icon https://materialdesignicons.com/icon/tray-full